tools/bin/mbedtls-kick-ci

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+set -eu
+
+script=$(basename $0)
+usage="$script: usage: $script PR_NUMBER [all|internal|internal-pr-head|internal-pr-merge|openci|...]"
+
+host_internal_ci=jenkins-mbedtls.oss.arm.com
+host_open_ci=mbedtls.trustedfirmware.org
+
+# ~/bin/jmt needs to contain the Jenkins user and API token for each CI, e.g.
+#
+# JENKINS_INTERNAL_USER_ID=user-id-for-internal-ci
+# JENKINS_INTERNAL_API_TOKEN=1111111111111111111111111111111111
+# JENKINS_OPENCI_USER_ID=user-id-for-open-ci
+# JENKINS_OPENCI_API_TOKEN=2222222222222222222222222222222222
+#
+# To get the user ID, go to Jenkins, e.g. https://mbedtls.trustedfirmware.org/,
+# click your name in the top right, then the Status page should show your Jenkins
+# user ID. Both internal and OpenCI user names will almost certainly be the same
+# as your GitHub ID.
+#
+# To get an API token, choose Configure from your user page, click Add new Token,
+# give it a name, then click Generate. You get one chance to copy it, so do it
+# now! Then put it into ~/bin/jmt.
+#
+# Make sure ~/bin/jmt is chmod 0600.
+
+. ~/bin/jmt
+
+kick_one () {
+    # kick_one DESC USER TOKEN URL
+    desc="$1"
+    user="$2"
+    token="$3"
+    url="$4"
+    echo "$desc\\c"
+    OUTPUT=$(curl -s -X POST --user $user:$token "$url")
+    if [ -z "$OUTPUT" ]; then
+        echo " started"
+    else
+        if echo "$OUTPUT" | fgrep -q "Error 404 Not Found"; then
+            echo " - PR $PR not found"
+            exit 1
+        else
+            echo " - FAILED: \\c"
+            echo "$OUTPUT" | perl -n -e 'print "$1\n" if /<title>(.*?)<.title>/;'
+            exit 2
+        fi
+    fi
+    did_something=1
+}
+
+if [ $# = 1 ]; then
```

Comment:
Technically this works, but it's a bit jarring: `=` is string equality, but it's ok because `$#` is guaranteed to be represented in decimal. The semantically correct way would be
```suggestion
if [ $# -eq 1 ]; then
```